networks.json

```
@@ -69,6 +81,10 @@
     "42161": {
       "address": "0xdaf33924925e03c9cc3a10d434016d6cfad0add5",
       "transactionHash": "0xea5a8c4e4fc3e69acedafd3ccee454c11952c7de8c76e37ab4b2b42fa2b1ebf4"
+    },
+    "232": {
+      "address": "0xdaF33924925E03C9cC3a10D434016D6cFad0aDd5",
```

Just to keep capitalization consistent:

```suggestion
      "address": "0xdaf33924925e03c9cc3a10d434016d6cfad0add5",
```

---

I just copied this from the explorer. Not sure why it has different capitalization.